dev/com.ibm.ws.microprofile.reactive.streams.operators_fat/test-applications/mprs.war/resources/log4j2.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This xml file results in: 
Stack Dump = javax.xml.stream.XMLStreamException: ParseError at [row,col]:[-1,-1]
Message: found: END_DOCUMENT, expected START_ELEMENT or END_ELEMENT